tests/existing-resources/version.tf

```diff
@@ -0,0 +1,9 @@
+terraform {
+  required_version = ">= 1.9.0"
+  required_providers {
+    ibm = {
+      source  = "IBM-Cloud/ibm"
+      version = "= 1.70.1"
```

Comment:
suggest to use >= 1.70.1 so the test will always use the latest provider

Reply:
Done.